File: sol/455.cpp
struct Solution {
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        int sptr = 0, res = 0;
        for (int &gg : g){
            while (sptr < s.size() && s[sptr] < gg){
                ++sptr;
            }
            if (sptr == s.size()){
                break;
            }
            ++res;
            ++sptr;
        }
        return res;
    }
};

File: sol/455/455.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        sptr = 0
        res = 0
        for gg in g:
            while sptr < len(s) and s[sptr] < gg:
                sptr += 1
            if sptr == len(s):
                break
            res += 1
            sptr += 1
        return res
